packit_service/worker/handlers/github_handlers.py

```diff
@@ -179,11 +201,27 @@ def run(self) -> TaskResults:
                 " to the issue comment.\n"
             )
 
-            self.project.create_issue(
+            issue = self.project.create_issue(
                 title=f"[packit] Propose update failed for release {self.data.tag_name}",
                 body=body_msg,
             )
 
+            if any(FILE_DOWNLOAD_FAILURE in error for error in errors.values()):
+                retry_message = (
+                    f"Since your archive was not found, we can't do the update in Fedora. "
+                    f"We will try in {PROPOSE_DOWNSTREAM_RETRY_TIME} minutes again. "
+                    f"If you haven't uploaded your archive yet, do it now, please."
+                )
+                issue.comment(body=retry_message)
```

Comment:
I'm thinking if this is not an overkill - would it make sense to only do a single comment if we run out of time?

My concern here is for p-s not to be too verbose.